packages/tideline/js/tidelinedata.js

```diff
@@ -207,16 +207,6 @@ function genRandomId() {
   return hexID.join('')
 }
 
-function getTimerFuncs() {
-  // To be able to enable it when needed:
-  if (false) { // eslint-disable-line no-constant-condition
```

Comment:
Why did you delete this? It was working

Reply:
my bad, I thought that a false condition always fails, but maybe it's ok in some other language ?

Reply:
Yes you are spot on! Tanguy script would support this kind of thing. :slightly_smiling_face: